client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -16,28 +16,26 @@ import QueryPlans from 'components/data/query-plans';
 
 const CALYPSO_JETPACK_CONNECT = '/jetpack/connect';
 
-const PlansLanding = React.createClass( {
+class PlansLanding extends Component {
 
-	propTypes: {
+	static propTypes = {
 		sites: React.PropTypes.object,
```

Comment:
This is a big part of the other fixes I mentioned. There a very weird `connect` `mapStateToProps` going on, that adds props that don't depend on `state` and may or may not be used. But yeah, I'd like to tackle all of that separately.